integration/creategetdelete_test.go

```diff
@@ -149,6 +149,41 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
+		Context("gitops apply", func() {
+			It("should add quickstart to the repo and the cluster", func() {
+				// Use a random branch to ensure test runs don't step on each others.
+				branch := namer.RandomName()
+				cloneDir, err := createBranch(branch)
+				Expect(err).ShouldNot(HaveOccurred())
+				defer deleteBranch(branch, cloneDir)
+
+				tempOutputDir, err := ioutil.TempDir(os.TempDir(), "gitops-repo-")
+				assertFluxManifestsAbsentInGit(branch)
+
+				deleteFluxInstallation(kubeconfigPath)
+				assertFluxPodsAbsentInKubernetes(kubeconfigPath)
+
+				eksctlSuccessWith(params{
+					Args: []string{"gitops", "apply",
+						"--git-url", Repository,
+						"--git-email", Email,
+						"--git-branch", branch,
+						// FIXME
```

Comment:
@marccarre The private ssh key in 1password doesn't work for me.

Reply:
After our discussion offline, are things working for you now?
(As also discussed with Martina, I've hopefully improved & clarified the instructions to setup the private SSH key in 1Password, in the notes for the test account.)

Reply:
Why not simply generate it on the fly and use https://github.com/stefanprodan/gitsrv ?

Reply:
It is working now. I think I would live the tests like this and try to set up gitsrv in a new PR that improves the IT

Reply:
Sounds good to me! (And thanks a lot for the pointer @2opremio! I wish I had known about it at the time 👍 )